website/docs/r/backup_compliance_policy.html.markdown

```diff
@@ -17,6 +17,15 @@ When enabled, the Backup Compliance Policy will be applied as the minimum backup
 
 -> **NOTE:** With Backup Compliance Policy enabled, cluster backups are retained after a cluster is deleted and backups can be used normally until retention expiration. When the Backup Compliance Policy is not enabled, Atlas deletes the cluster's associated backup snapshots when a cluster is terminated. By default, a Backup Compliance Policy is not enabled. For more details see [Back Up, Restore, and Archive Data](https://www.mongodb.com/docs/atlas/backup-restore-cluster/). 
 
+-> **NOTE:** If encountering errors when deleting an Atlas Cluster which has an associated `mongodbatlas_cloud_backup_schedule` resource. We suggest the following: 
+* `mongodbatlas_cloud_backup_schedule` fails during delete because an existing backup compliancy policy is enabled. This error is expected.
```

Comment:
```suggestion
* `mongodbatlas_cloud_backup_schedule` fails when you delete an Atlas cluster because an existing backup compliancy policy is enabled, which is expected.
```
Suggestion: I'd maybe put this as the first sentence in the NOTE since it seems out of place to say this is an expected error when I'm expecting to read some suggestions, and it seems that each suggestion following builds off this expected error